src/components/Table/Table.less

```diff
@@ -182,7 +189,7 @@
 				.@{prefix}-Table-Th-inner-resize {
 					cursor: ew-resize;
 					width: 3px;
```

Comment:
I tried adding a negative right margin so that the resizer was centered over the border.  Worked in FF, but the hover states were weird in Chrome.